Detectors/Upgrades/ITS3/workflow/src/TrackerSpec.cxx

```diff
@@ -73,8 +73,27 @@ void TrackerDPL::init(InitContext& /*ic*/)
     trackParams[2].TrackletMinPt = 0.1f;
     trackParams[2].CellDeltaTanLambdaSigma *= 4.;
     trackParams[2].MinTrackLength = 4;
+
     LOG(info) << "Initializing tracker in async. phase reconstruction with " << trackParams.size() << " passes";
 
+  } else if (mMode == "async_misaligned") {
+
+    trackParams.resize(3);
+    trackParams[0].PhiBins = 32;
+    trackParams[0].ZBins = 64;
+    trackParams[0].CellDeltaTanLambdaSigma *= 3.;
+    trackParams[0].SystErrorZ2[0] = 1.e-4;
+    trackParams[0].SystErrorZ2[1] = 1.e-4;
+    trackParams[0].SystErrorZ2[2] = 1.e-4;
+    std::copy(trackParams[0].SystErrorZ2.begin(), trackParams[0].SystErrorZ2.end(), trackParams[0].SystErrorY2.begin());
+    trackParams[0].MaxChi2ClusterAttachment = 60.;
+    trackParams[0].MaxChi2NDF = 40.;
+    trackParams[1] = trackParams[0];
+    trackParams[2] = trackParams[0];
+    trackParams[1].MinTrackLength = 6;
+    trackParams[2].MinTrackLength = 4;
+    LOG(info) << "Initializing tracker in misaligned async. phase reconstruction with " << trackParams.size() << " passes";
+
```

Comment:
Just so you know, in ITS, we have three reconstruction options: sync, async, and cosmics.
Any overriding of the selections is done via configkeyvals.

Reply:
Thanks for taking the time.
 yes, these are more less temporary right now.